docs/reference_guide.md

```diff
@@ -3092,6 +3093,25 @@ SRC 18:C0:4D:08:2E:BB, DST 74:83:C2:7F:8C:FF
 ^C
 ```
 
+## 29. `cgroup_path`: Convert cgroup id to cgroup path
+
+Syntax: `cgroup_path(int cgroupid, string filter)`
+
+Converts the given cgroup id into the corresponding cgroup path for each cgroup hierarchy the id
+appears in. Because the conversion is done in user space, the resulting object can only be used for
+printing.
+
+Optionally a string literal may be passed as the second argument to filter cgroup hierarchies to look
+in (interpreted as a regex).
```

Comment:
I'm wondering how complex we expect the regex expressions to be. If it's mostly about using `.*`, then it would be better to allow just `*` as a placeholder for a sequence of any chars (instead of full regexes) as we do in other places (e.g. probe matcher). You could use the `wildcard_match` function then.

Reply:
I don't expect them to be used too much, most of the time the user will probably want to use either `.*` or an exact name of the hierarchy. `wildcard_match` supports only `*`, though, and not something like `(unified|cpu)`, unlike the regex.

Reply:
I believe that we'll eventually get a support for that into our matcher so I'd still prefer going with `wildcard_match` for now. Users are also more used to specify path patterns using plain `*`, I'd say.

Reply:
OK, I'll move `get_tokens` to utils.cpp (currently it's private to probe_matcher.cpp) and swap the regex match with `wildcard_match` (which can be extended in another PR). I'd normally use glob(3) to match directly against the path, but cgroup2 can be mounted directly into `/sys/fs/cgroup`, which would break it...